sdk/src/account_utils.rs

```diff
@@ -1,11 +1,13 @@
 //! useful extras for Account state
 use crate::account::{Account, KeyedAccount};
 use crate::instruction::InstructionError;
-use bincode::ErrorKind;
 
 /// Convenience trait to covert bincode errors to instruction errors.
 pub trait State<T> {
     fn state(&self) -> Result<T, InstructionError>;
+    //    fn with_state<R, E, F>(&mut self, func: F) -> Result<Result<R, E>, InstructionError>
```

Comment:
unused?

Reply:
I am holding off putting these in, because with account.rs support, this file actually goes away...